### PR TITLE
Solving RESTWS-713

### PR DESCRIPTION
**Pull request by fruether**
Adding new attribute to set of return values
Create one simple test case to make sure everything is working
https://issues.openmrs.org/browse/RESTWS-713

### CODE REVIEW

**Comment by fruether** [comment]
Would it be possible to change the reference Person object so that it contains actually a causeofDeath non coded ?

**Comment by fruether** [comment]
I have to add methods like getCreatableProperties (if I understood the example correctly) so the PR is not ready yet

File: omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_11.PersonResource1_11;
+
+/**
+ * {@link Resource} for Person, supporting standard CRUD operations
+ */
+@Resource(name = RestConstants.VERSION_3 + "/person", order = 1, supportedClass = Person.class, supportedOpenmrsVersions = {
```
Comment (wluyima) [inline omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java]: I doubt if order adds value here? Given that it's the same value anyways like PersonResource1_11

Comment (wluyima) [inline omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java]: Should be version 1, this is rest API and not a resource version.

File: omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestConstants.java [hunk 2]
```diff
@@ -73,6 +73,11 @@
 	 */
 	public static final String VERSION_2 = "v2";
 	
+	/**
+	 * The version number for the third rest web services representations
+	 */
+	public static final String VERSION_3 = "v3";
```
Comment (wluyima) [inline omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestConstants.java]: You can remove this

**Pull request closed without merging**